### PR TITLE
fix: fcp matching can't work

### PR DESCRIPTION
**Pull request by pauldambra**
we match FCP measurements onto navigation events using the pageview_id

but we don't send a pageview_id any more, so that can't work

we can assume that an FCP is for the last navigation event

@coderabbitai ignore

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts [hunk 1]
```diff
@@ -999,3 +987,26 @@ function deduplicatePerformanceEvents(events: PerformanceEvent[]): PerformanceEv
         })
         .reverse()
 }
+
+/**
+ * If we have paint events we should add them to the appropriate navigation event
+ * this makes it easier to draw performance cards for navigation events
+ */
+function matchPaintEvents(performanceEvents: PerformanceEvent[]): PerformanceEvent[] {
+    // KLUDGE: this assumes that the input is sorted by timestamp and relies on the identity of the events to mutate them
```
Comment (daibhin) [inline frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts:996]: Is there any reason we wouldn't do the sorting inside this function to make sorting a requirement

**Review by daibhin — APPROVED** [review]
One comment but doesn't have to block you

File: frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts (hunk 1 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts:996]: yeah, I was torn... was too lazy to find out if it is cheap to sort an already sorted thing....

Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts]: turns out it is slightly faster to sort an already sorted list of timestamps... but tbh despite the wastefulness of sorting multiple times a quick benchmark managed 38k sort operations per second for a list of 1000 items so I'm completely worrying unnecessarily 🤣

**Pull request merged**